packages/snaps-execution-environments/package.json

```diff
@@ -68,6 +68,7 @@
     "eslint-plugin-jsdoc": "^39.6.2",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.2.1",
+    "http-server": "^14.1.1",
```

Comment:
What is this for?

Reply:
I used this for testing, but forgot to remove it. 😅